modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/integration/v1_1/GeoServerIntegrationTest.java

```diff
@@ -113,7 +113,7 @@ public void testInfo() throws IOException {
         assertTrue(info.getKeywords().contains("sfRoads roads"));
         assertEquals("roads_Type", info.getTitle());
         assertEquals("http://www.openplans.org/spearfish", info.getSchema().toString());
-        assertEquals(589275.24, info.getBounds().getMinX(),0.01);
+        assertEquals(589242.03, info.getBounds().getMinX(),0.01);
```

Comment:
Is this in meters/feet?  Hmm...  I'm not sure what to do suggest regarding the change in precision.